lib/letters.ml

```diff
@@ -205,8 +214,15 @@ let send ~config:c ~recipients:r ~message:m =
         | Error _ -> failwith "Config hostname is not valid hostname"
         | Ok hostname -> hostname )
   in
+  let* detected_cert = Ca_certs.detect () in
+  let path =
+    match (c.ca_dir, detected_cert) with
+    | Some dir, _ -> dir
+    | None, Some (`Ca_file dir) -> dir
+    | None, None -> failwith "No CA certificates directory provided or found"
+  in
   let* certs =
-    match Fpath.of_string c.ca_dir with
+    match Fpath.of_string path with
     | Ok path -> certs_of_pem_directory ~ext:"pem" path
```

Comment:
Does this really work? I'm a bit puzzled what the `path` ends up being and how this `certs_of_pem_directory` works when our detected files are actually CA bundles and not the CA path while here we only read in all `pem` files.